README.md

```diff
@@ -139,6 +139,13 @@ If you are using a Ubuntu/Debian system, install the following packages:
 
 The installation command for both systems can be found [here](./dashboards-reports/rendering-engine/headless-chrome/README.md).
 
+### Download Error: 
+
+If you see Error: net::ERR_ADDRESS_INVALID in the logs while downlaoding report, add the following configurations in opensearch_dashboards.yml
+```
+opensearch_reporting.osd_server.hostname: '127.0.0.1'
```

Comment:
can this default to 127.0.0.1? this troubleshooting should only be needed for users who are connecting to a remote endpoint